README.md

```diff
@@ -53,6 +53,13 @@ npm run compile
 code ./vscode-workspaces/multiroot-simple.code-workspace
 ```
 
+OR
+
+```
+From the lightning-language-server project, open the file 'multiroot-simple.code-workspace'

```

Comment:
I was having issues with the above `code ./vscode-workspaces/multiroot-simple.code-workspace` command. This other methods works just as well.